src/app/app.module.ts

```diff
@@ -1,8 +1,8 @@
+import { HttpClient, HttpClientModule } from '@angular/common/http';
 import { NgModule } from '@angular/core';
-import { ApiModule } from './api/generated/api.module';
 import { BrowserModule } from '@angular/platform-browser';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
-import { HttpClient, HttpClientModule} from '@angular/common/http';
+import { ApiModule } from './api/generated';
```

Comment:
Hmm, geen idee waarom deze in dit PR zit, maar mij lijkt dat we 'm wel nodig hebben.